### ISSUE TITLE
Fine-tuning the contrasts of colors of A+ for better accessibility

### ISSUE THREAD

**Issue by saarasat** [apluslms/a-plus#568]
At the moment, some of the colors used in A+ UI do not have strong enough contrast-ratio, for them to pass the AA-level of [web content accessibility guidelines ](https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum). The contrast-ratio can be assessed at: https://contrast-ratio.com/

In order to fix this issue, some of the colors present in A+ should be fine-tuned (eg. made slightly darker or lighter) to meet the AA-level standards, for instance the MOOC-badge here:

![image](https://user-images.githubusercontent.com/43036333/84268484-646f0a00-ab30-11ea-9efe-f7b0f6422749.png).

**Comment by saarasat** [comment]
Alright, I'll then fix those badges, which do not use the font color selection logic.
